### PR TITLE
define manifest data type for FIP 0031

### PR DESCRIPTION
**Pull request by vyzo**

### CODE REVIEW

File: FIPS/fip-0031.md [hunk 1]
```diff
@@ -142,8 +142,8 @@ struct Manifest {
 }
 ```
 
-The _Manifest_ data for version 1 (the only version currently defined) is a Cid linking
-to an IPLD `Vector<(String, Cid)>`.
+The _Manifest_ data for version 1 (the only version currently defined) is a CID linking
+to an IPLD `Vector<(String, CID)>`.
```
Comment (raulk) [inline FIPS/fip-0031.md:146]: We should formally define the order of this vector.

Comment (raulk) [reply FIPS/fip-0031.md:146]: Here and, most importantly, in the system actor's state.

Comment (vyzo) [reply FIPS/fip-0031.md:146]: So whats the order? By type? Lexicographic?
I dont think it really matters all that much, but i am not opposed ti specifying it either.

Comment (vyzo) [reply FIPS/fip-0031.md:146]: So the bundler uses a BTreeMap, which orders by actor type -- this is perfectly reasonably I guess, should we make this the standard order?

Comment (vyzo) [reply FIPS/fip-0031.md]: Updated to define the order by enum position.

**Pull request merged**